GOVERNANCE.md

```diff
@@ -0,0 +1,105 @@
+# io.js Project Governance
+
+## Working Group
+
+The io.js docker project is jointly governed by a Working Group (WG)
+which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+[README.md](./README.md#people).
+
+## Collaborators
+
+The [iojs/docker-iojs](https://github.com/iojs/docker-iojs) GitHub repository is
+maintained by the WG and additional Collaborators who are added by the
+WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These
+individuals are identified by the WG and their addition as
+Collaborators is discussed as a pull request to this project's
+[README.md](./README.md#people).
+
+_Note:_ If you make a significant contribution and are not considered
+for commit-access log an issue or contact a WG member directly.
+
+Modifications of the contents of the iojs/docker-iojs repository are made on
+a collaborative basis. Anybody with a GitHub account may propose a
+modification via pull request and it will be considered by the project
+Collaborators. All pull requests must be reviewed and accepted by a
+Collaborator with sufficient expertise who is able to take full
+responsibility for the change. In the case of pull requests proposed
+by an existing Collaborator, an additional Collaborator is required
+for sign-off. Consensus should be sought if additional Collaborators
+participate and there is disagreement around a particular
+modification. See _Consensus Seeking Process_ below for further detail
+on the consensus model used for governance.
+
+Collaborators may opt to elevate significant or controversial
+modifications, or modifications that have not found consensus to the
+WG for discussion by assigning the ***WG-agenda*** tag to a pull
+request or issue. The WG should serve as the final arbiter where
+required.
```

Comment:
I'm not thinking this will ever be an issue, but out of curiosity, how will this work in practice when there are no WG meetings? Will the issue be put on hold unit all WG-members have commented? In such a case, will that be contradictory to the consensus seeking process?